vars/unitTestPost.groovy

```diff
@@ -87,9 +87,9 @@ def call(Map config = [:]) {
     def valgrind_pattern = config.get('valgrind_pattern', '*.memcheck.xml')
     stash name: config['valgrind_stash'], includes: valgrind_pattern
   }
-  def no_record_issues = config.get('no_record_issues', true)
-  if ((!stage_info['with_valgrind'] && no_record_issues) || (stage_info['NLT'] == 1)) {
-    cb_result = currentBuild.result
+  def record_issues = config.get('record_issues', true)
+  if ((!stage_info['with_valgrind'] && record_issues) || stage_info['NLT']) {
+    def cb_result = currentBuild.result
```

Comment:
I think @JohnMalmberg mentioned that you have scoped `cb_result here` to just this `if` block and it will not be available where you use it below.

Reply:
I thought I'd fixed that, it's used twice and in scope for just where required both times.  Then again I've not done much groovy.

Reply:
Hrm.  Maybe you did and it's just all of the leading whitespace in the side-by-side diff that is making me think they are not in the same scope.